recipes/ixwebsocket/all/conanfile.py

```diff
@@ -0,0 +1,147 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class IXWebSocketConan(ConanFile):
+    name = "ixwebsocket"
+    description = "WebSocket client/server"
+    topics = ("conan", "IXWebSocket", "socket", "websocket")
+
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/machinezone/IXWebSocket"
+
+    # License for the library
+    license = "BSD-3-Clause"
+
+    exports_sources = ["CMakeLists.txt"]
+
+    settings = "os", "compiler", "build_type", "arch"
+
+    short_paths = True
+    generators = "cmake"
+
+    options = {
+        "use_mbed_tls": [False, True],
+        "use_tls": [True, False],
+        "use_openssl": [False, True],
+        "use_vendored_third_party": [True, False],
+        "use_ws": [False, True],
```

Comment:
This option requires libjsoncpp and spdlog

Reply:
The last one? It's also a CLI tool actually. No idea why I added that option, so I'm just gonna exclude it

Reply:
Yes, `use_ws`